File: Python/arrays/search_number_over_limit.py
# coding=utf8
# Declaração
vetor = []
maior50 = False

for i in range(0, 10):
    num = int(input("Informe o número {0}/10 para o vetor: ".format(i + 1)))
    vetor.append(num)
    
for i in vetor:
    if i > 50:
        print("\nO número {0} está na posição {1}/10".format(i, vetor.index(i) + 1))
        maior50 = True

if maior50 == False:
    print("\nNenhum número acima de 50 foi digitado")

File: C/Algoritmos_e_logica_de_programacao/arrays/search_number_over_limit.c
#include <stdio.h>

int main() {
	//Declaração
	int vetor[10];
	char maior50 = 0;

	//Entrada
	for (int i = 0; i < 10; i++) {
		printf ("Informe o número %d/10 para o vetor: ", (i + 1));
		fflush (stdout);
		scanf ("%d", &vetor[i]);
	}

	//Processamento
	for (int i = 0; i < 10; i++) {
		if (vetor[i] > 50) {
			printf ("O número %d está na posição %d\n", vetor[i], (i+1));
			maior50 = 1;
		}
	}

	if (maior50 < 1) {
		printf ("Nenhum número acima de 50 foi digitado.");
	}
}
